src/App.css

```diff
@@ -1,3 +1,310 @@
 .App {
   font-family: Arial, Helvetica, sans-serif;
 }
+
+.navBar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 10px;
+  background-color: #fff;
+}
+
+.projectHeading {
+  width: auto;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.175rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.menuitem {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 1;
+  text-decoration: none;
+}
+
+.bookContainer {
+  background-color: #f5f6fa;
+  margin: 10px;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```

Comment:
- [ ] As the previous reviewer mentioned please try to stick with the Zeplin design in your book cards as well. let's fix the background color and spacing of the elements to match the Zeplin design.


Zeplin Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/202866027-fc74a866-8cad-4d6b-8bb8-123075f894af.png) |  ![image](https://user-images.githubusercontent.com/60618877/202865996-6e99f979-3fac-4ed9-8fdf-0065b6248c67.png)